enterprise/frontend/src/metabase-enterprise/sandboxes/index.js

```diff
@@ -53,11 +53,9 @@ const getEditSegmentedAccessPostAction = (entityId, groupId, view) =>
   push(getEditSegementedAccessUrl(entityId, groupId, view));
 
 if (hasPremiumFeature("sandboxes")) {
-  PLUGIN_ADMIN_USER_FORM_FIELDS.push({
-    name: "login_attributes",
-    title: t`Attributes`,
-    type: LoginAttributesWidget,
-  });
+  PLUGIN_ADMIN_USER_FORM_FIELDS.FormLoginAttributes = LoginAttributesWidget;
```

Comment:
This changed from being a list of "form field" objects, to just being a single component